docs/source/examples/efficient_dask.ipynb

```diff
@@ -0,0 +1,654 @@
+{
```

Comment:
<p>Don't forget to organize this into an appropriate folder and subheader in examples. Maybe also link to the documentation with some comments here about what <code>dask</code>  is. That it enables out-of-memory and parallel computation for large datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I am unsure how to put those into an ordering. Would you mind renaming and structuring those?

Reply:
ah. now I get it.